pheps/phep-0001.md

```diff
@@ -0,0 +1,338 @@
+
+11. **Open Issues** - While a PHEP is in draft, ideas can come up which warrant further discussion. Those ideas should be recorded so people know that they are being thought about but do not have a concrete resolution. This helps make sure all issues required for the PHEP to be ready for consideration are complete.
+
+12. **Footnotes** - A collection of footnotes cited in the PHEP, and a place to list non-inline hyperlink targets.
+
+13. **Copyright/license** - Each new PHEP must be placed under a dual license of public domain and [CC0-1.0-Universal](https://creativecommons.org/publicdomain/zero/1.0/) and include citation information (see this PHEP for an example).
+
+# PHEP Formats and Templates
+<a name="phep-formats-and-templates"></a>
+PHEPs are UTF-8 encoded text files using the Markdown format. A future PHEP may include a template. PHEP Markdown must be compatible with the most recent version of the [CommonMark specification](https://spec.commonmark.org/) at the time the PHEP is approved. PHEPs must not use [GitHub extensions](https://github.github.com/gfm/).
+
+The editors are responsible for enforcing PHEP format and consistency of style within a PHEP. They may use automated tools (e.g. pre-commit hooks) to assist.
+
+# PHEP Header Preamble
+<a name="phep-header-preamble"></a>
+Each PHEP must begin with an [RFC 2822](https://datatracker.ietf.org/doc/html/rfc2822.html) style header preamble, marked off with a [code fence](https://spec.commonmark.org/current/#fenced-code-blocks). The headers must appear in the following order. Headers marked with "*" are optional. All other headers are required.
+
+      ```
+      PHEP: <phep number>
+      Title: <phep title>
+      Author: <list of authors' real names and optionally, email addrs>
+      Discussions-To: <URL of current canonical discussion thread>
+      Status: <Draft | Accepted | Rejected |
+               Withdrawn | Final | Superseded>
+      Type: <Standards Track | Informational | Process>
+      Content-Type: text/markdown; charset=UTF-8; variant=CommonMark
+    * Requires: <phep numbers>
+      Created: <date created on, in dd-mmm-yyyy format>
+      Post-History: <dates, in dd-mmm-yyyy format,
+                     inline-linked to PHEP discussion threads>
+    * Replaces: <phep number>
+    * Replaced-By: <phep number>
+    * Resolution: <url>
+      ```
+
+The Author header lists the names and optionally the email addresses and/or [ORCIDs](https://orcid.org) of all the authors of the PHEP. The format of the Author header values must be:
+
+`Random J. User <random@example.com> <https://orcid.org/0000-0000-0000-0000>`
```

Comment:
I am not thrilled by this formatting. We could try the URL without the angle brackets or make it a full MD link, or just call this good enough.

Reply:
Hmmm that's tricky. I don't dislike the way it is now. Just a question of which option looks best. Links could be nice too, but it might look odd if they're links while e.g. the "Discussions-To" isn't. 🤔

Reply:
Nothing saying `Discussions-To` and other headers can't be updated, too. e.g. NEP-0 uses rST fields for the preamble but, well, they're using rST instead of Markdown. (PEP-1 was also rST but I switched to MD because of the better support in GitHub and nobody objected at the meeting--that's open to revisiting as well, of course! Everything is!)

Reply:
True, true. I do think MD was the right call over rST. And I feel like the way we have it now requires less typing than making it a full MD link. I'm tempted to leave it.

Reply:
MD links are nicer looking, but in the grand scheme of things, I agree with Shawn that what was have here seems good enough. We're already requesting a lot of typing as is..